packages/tasks_repository/lib/src/models/task.dart

```diff
@@ -2,5 +2,5 @@ class Task {
   String description;
   bool isDone;
 
-  Task(this.description, this.isDone);
+  Task({required this.description, required this.isDone});
```

Comment:
small tip that your IDE analyzer should suggests you: move constructor before other members, in this case constructor should be before all final properties.